views/layout.hbs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles/style.css">
```

Comment:
About linking different `css` files. 
You can link each file inside the `hbs` page that you want it to be applied, instead of linking it in the `layout.hbs`.